docs/design/eth-subscribe.md

```diff
@@ -38,12 +38,18 @@ wscat -c wss://<env>.hashio.io/api
 
 // then call subscription
 {"jsonrpc":"2.0","id": 1, "method": "eth_subscribe", "params": [SUBSCRIPTION_TYPE, PARAMS]}
+
+// call to newHeads
+{"id":1,"jsonrpc":"2.0","method":"eth_subscribe","params":["newHeads"]}
 ```
 
 #### Response
 
 ```javascript
 {"jsonrpc":"2.0","id": 1, "result": SUBSCRIPTION_ID}
+
+// newHeads
+{"jsonrpc":"2.0","id": 1, "result": SUBSCRIPTION_ID}
```

Comment:
Should we show some example of `SUBSCRIPTION_ID`, like Geth does https://geth.ethereum.org/docs/interacting-with-geth/rpc/pubsub?